src/main/scala/com/amazon/deequ/VerificationResult.scala

```diff
@@ -70,6 +77,23 @@ object VerificationResult {
       "constraint_status", "constraint_message")
   }
 
+  /**
+   * For each check in the verification suite, adds a column of row-level results
+   * to the input data if that check contains a column.
+   *
+   * Accepts a naming rule
+   */
+  def toRowLevelResults(
```

Comment:
Minor: Can we name this in accordance with the other public methods? `rowLevelResultsAsDataFrame` perhaps?